ci-operator/config/openshift/csi-operator/openshift-csi-operator-master.yaml

```diff
@@ -294,6 +306,24 @@ tests:
     env:
       TEST_SCENARIOS: \[CSI-Driver\]|\[Azure-File-CSI-Driver\]|\[Azure-File-CSI-Driver-Operator\]
     workflow: openshift-e2e-azure-csi-extended
+- as: e2e-openstack
+  optional: true
+  run_if_changed: ^(Dockerfile\.openstack-cinder|legacy/openstack-cinder-csi-driver-operator/.*)
+  steps:
+    cluster_profile: openstack-vexxhost
+    env:
+      BASE_DOMAIN: shiftstack.devcluster.openshift.com
+      CONFIG_TYPE: minimal
+      TEST_SUITE: experimental/reliability/minimal
+    workflow: openshift-e2e-openstack-ipi
+- as: e2e-openstack-cinder-csi
+  run_if_changed: ^(Dockerfile\.openstack-cinder|legacy/openstack-cinder-csi-driver-operator/.*)
```

Comment:
nice. this would be useful for other operators too!

Reply:
It may need to be a bit more involved, since conceivably any chance to the "common" code could affect an individual driver. However, we could just rely on the fact that (a) changes to the asset generation will require an asset generation run (I assume there's a job to check that - I have yet to look) and (b) if it's a programming error the build will likely fail. In any case, a can to kick down the road slightly until we start reworking this driver to take advantage of the generator.